src/main/java/umc/spring/web/controller/StoreController.java

```diff
@@ -0,0 +1,68 @@
+package umc.spring.web.controller;
+
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.Parameters;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+import umc.spring.Vaildator.annotation.CheckPage;
+import umc.spring.apiPayload.ApiResponse;
+import umc.spring.repository.ReviewRepository.ReviewRepository;
+import umc.spring.service.ReviewService.ReviewQueryServiceImpl;
+import umc.spring.service.StoreService.StoreQueryServiceImpl;
+import umc.spring.web.controller.dto.ReviewDTO.ReviewRequestDTO;
+import umc.spring.web.controller.dto.ReviewDTO.ReviewResponseDTO;
+import umc.spring.web.controller.dto.StoreDTO.StoreRequestDTO;
+import umc.spring.web.controller.dto.StoreDTO.StoreResponseDTO;
+
+@RestController
+@RequiredArgsConstructor
+@Validated
+@RequestMapping("/stores")
+public class StoreController {
+
+    private final StoreQueryServiceImpl storeQueryService;
+
+
+    @PostMapping("{regionId}")
+    public ApiResponse<StoreResponseDTO.JoinResultDTO>
+    join(
+            @PathVariable Long regionId,
+            @RequestBody @Valid StoreRequestDTO.joinStoreDTO request
+    ) {
+        StoreResponseDTO.JoinResultDTO result = storeQueryService.joinStore(regionId, request);
+        return ApiResponse.onSuccess(storeQueryService.joinStore(regionId, request));
```

Comment:
현재 `storeQueryService.joinStore(regionId, request)`를 두번 호출하고 있네요!!! 중복 호출은 성능 저하나 오류를 유발할 수도 있으니 다음과 같이 수정하면 좋을 거 같아요!!!
```suggestion
        StoreResponseDTO.JoinResultDTO result = storeQueryService.joinStore(regionId, request);
        return ApiResponse.onSuccess(result));
```